react-common/components/palette/ColorPickerField.tsx

```diff
@@ -25,6 +25,7 @@ export const ColorPickerField = (props: ColorPickerFieldProps) => {
     }
 
     const onTextInputChanged = (newValue: string) => {
+        newValue = newValue.trim();
```

Comment:
Just to confirm, this is the fix for what I mentioned yesterday where it was ignoring input if you typed in ` #ffffff `, right?

Reply:
yes thanks for catching it!